x-pack/plugins/observability/public/components/app/section/uptime/index.tsx

```diff
@@ -24,34 +24,45 @@ import { SectionContainer } from '../';
 import { getDataHandler } from '../../../../data_handler';
 import { useChartTheme } from '../../../../hooks/use_chart_theme';
 import { FETCH_STATUS, useFetcher } from '../../../../hooks/use_fetcher';
+import { useHasData } from '../../../../hooks/use_has_data';
+import { useTimeRange } from '../../../../hooks/use_time_range';
 import { Series } from '../../../../typings';
 import { ChartContainer } from '../../chart_container';
 import { StyledStat } from '../../styled_stat';
 import { onBrushEnd } from '../helper';
 
 interface Props {
-  absoluteTime: { start?: number; end?: number };
-  relativeTime: { start: string; end: string };
   bucketSize?: string;
 }
 
-export function UptimeSection({ absoluteTime, relativeTime, bucketSize }: Props) {
+export function UptimeSection({ bucketSize }: Props) {
   const theme = useContext(ThemeContext);
+  const chartTheme = useChartTheme();
   const history = useHistory();
+  const { forceUpdate, hasData } = useHasData();
+  const { rangeFrom, rangeTo, absStart, absEnd } = useTimeRange();
```

Comment:
nit: for consistency:
```suggestion
  const { relativeStart, relativeEnd, absStart, absEnd } = useTimeRange();
```